_docs/applications/mssql.md

```diff
@@ -42,52 +46,28 @@ Before you start, please ensure you have StorageOS installed and ready on a Linu
 
    ```
    $ sudo docker logs mssql | more
-   Configuring Microsoft(R) SQL Server(R)...
-   Configuration complete.
-   paltelemetry: Loading the directory /var/opt/mssql/.system/.system/telemetry failed. Errno [2]
-   This is an evaluation version.  There are [172] days left in the evaluation period.
-   2017-01-26 16:13:33.63 Server      Microsoft SQL Server vNext (CTP1.2) - 14.0.200.24 (X64)
-       Jan 10 2017 19:15:28
-       Copyright (C) 2016 Microsoft Corporation. All rights reserved.
-       on Linux (Ubuntu 16.04.1 LTS)
+   ...
+   2017-04-13 09:24:22.28 Server      Dedicated admin connection support was established for listening locally on port 1434.
+   2017-04-13 09:24:22.28 spid17s     SQL Server is now ready for client connections. This is an informational message; no user action is required.
    ```
 
-## Install MS SQL Tools on Ubuntu
-
-Sqlcmd is part of the SQL Server command-line tools, which are not installed automatically with SQL Server on Linux and not present in the container image.
+## Install SQL Server command-line tools on Linux
 
-1. Install MS SQL Server Xenial CLI Package
+Sqlcmd is now included as part of the SQL Server container however command-line tools and can be installed separately.
 
-   ```
-   $ curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
-   $ curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list | sudo tee /etc/apt/sources.list.d/msprod.list
-   $ sudo apt-get update
-   $ sudo apt-get install mssql-tools unixodbc-dev
-   ```
+Depending on your Linux distribution, the installation will vary - for more details please consult the [Microsft's website](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-setup-tools).
 
-2. Add Symbolic Links
-
-   ```
-   $ sudo ln -sfn /opt/mssql-tools/bin/sqlcmd-13.0.1.0 /usr/bin/sqlcmd
-   $ sudo ln -sfn /opt/mssql-tools/bin/bcp-13.0.1.0 /usr/bin/bcp
-   ```
-
-3. Optional: Add MSSQL Tools to your Path
+## Create a Test Database
 
-   ```
-   $ echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bash_profile
-   $ echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bashrc
-   $ source ~/.bashrc
-   ```
+1. Connect to SQL Server
 
-## Create a Test Database
+   >**Note**: We are using *localhost* in this example - if you are running this from a remote host you will need to specify the target server IP.
```

Comment:
According to https://docs.microsoft.com/en-us/sql/tools/sqlcmd-utility, -S localhost is the default. If it's not needed then remove it and move the note to line 104.